host-provision.sh

```diff
@@ -47,6 +47,14 @@ eval set -- "$options"
 
 while true; do
     case "$1" in
+    --u01-lun)
```

Comment:
I thin we'll need some documentation for this new command and parameters.  In either this PR or another, can we write up the usage and purpose of host-provision.sh, including example invocation, somewhere in docs/?  Thanks!

Reply:
Absolutely, I was thinking of a markdown page later down the line.
Will add it with a schematic along with a date disclaimer (as it can get outdated quickly) and also will include the usage details.